src/examples/test-and-release.yml

```diff
@@ -0,0 +1,39 @@
+# Copyright (c) 2018, salesforce.com, inc.
+# All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+# For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+
+description: >
+  Test npm package agaisnt node@12 and node@latest and then publish the package to npm
+usage:
+  version: 2.1
+  orbs:
+    release-management: salesforce/npm-release-management@x.y.z
+
+  workflows:
+    version: 2
+    test-and-release:
+      jobs:
+        - release-management/test-package:
+            name: node-latest
+            node_version: latest
+        - release-management/test-package:
+            name: node-12
+        - release-management/release-package:
+            context: AWS
+            salesforce_key: $SALESFORCE_KEY
+            npm_token: $NPM_TOKEN
+            key_path: /root/salesforce-cli.key
+            requires:
+              - node-latest
+            filters:
+              tags:
+                only: /^v.*/
+              branches:
+                ignore: /.*/
+        - release-management/cache:
```

Comment:
As mentioned in the config plugin, I think we should remove this and handle cache for them behind the scenes by just doing it after the install.